types/node-zopfli/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "no-outside-dependencies": false
```

Comment:
Why do you need this? If I remember right, this lint error can be fixed by adding `/// <reference types="node" />`. (Though I could be misremembering)

Reply:
Sorry about that.

When running tests locally, I was seeing many errors like
```
Error: /Users/kyle.turney/dev/DefinitelyTyped/node_modules/@types/node/assert.d.ts:1:1
ERROR: 1:1  no-outside-dependencies  File /Users/kyle.turney/dev/DefinitelyTyped/node_modules/@types/node/assert.d.ts comes from a `node_modules` but is not declared in this type's `package.json`.  See: https://github.com/Microsoft/dtslint/blob/master/docs/no-outside-dependencies.md
```

Modifying `tslint.json` was the first fix I encountered while searching the repo.
Turns out the problem was actually that I had not done a `git sparse-checkout add types/node`.
I guess I glossed over the full meaning of `git sparse-checkout add types/<type> types/<dependency-type> ...` in https://github.com/DefinitelyTyped/DefinitelyTyped#partial-clone